pages/en/contrib/style-guide.md

```diff
@@ -24,6 +25,152 @@ Style conventions for this document:
 
 ## General guidelines
 
+{% include important.html content="
+
+The rules described in this document are evolving, therefore not all of our
+code-base is following them yet. This is expected and OK. Our approach is to
+incrementally improve the parts that are changing most often, i.e. fix the
+coding style only as part of changes needed for a bug fix/new feature.
+
+" %}
+
+### Variable declarations
+
+Prefer to use immutable variables declared with `const` keyword:
```

Comment:
Might be worth pointing out that `const` makes the reference immutable but not the referenced object.  It's something that seems to trip up a lot of people.